### PR TITLE
Several refinements in configServer

### LINKED ISSUE #109: Uploading tracks auf Unterseite mit Infotext
Aktuell klickt man auf Upload und der Browser lädt und lädt und ohne Interaktion werden alle Tracks hochgeladen.

Kann man nicht erst auf eine neue Seite verlinken, wo das zumindest kurz beschrieben ist, was gerade passiert und dass man einfach warten soll?

### LINKED ISSUE #167: Change in Config-Menu
The "Upload Tracks" is in the chapter "Maintenance".
I'd say it's not a maintenance-task, but would prefer to create an own chapter "Upload" (or even a better name? Suggestions welcome). And, because it's probably the most used function in the near future, put it on the first place of the config menue.

### PR DESCRIPTION
**Pull request by amandel**
- avoid duplicate content in const strings
- move style to header for faster rendering
- Add a title to each page (#141)
- "Stream" upload page while uploading
- Show upload results in upload page (fixes #109)
- "Stream" SD directory page
- Add content-type for cvs in download
- Refine `/` handling in SD download
- Add icons to SD file browser
- back arrow calls history.back()
- html cleanup
- read GPS always and early in config mode
- add system info page
- Place "Upload Track" at top (fixes #167)
- Collect TOF sensor statistics and display them on the about page

There are other things todo but this is already a large PR.

### CODE REVIEW

File: platformio.ini [hunk 1]
```diff
@@ -31,13 +31,11 @@ board_build.partitions = min_spiffs.csv
 lib_deps =
     ; https://arduinojson.org/v6/api/
     bblanchon/ArduinoJson @ ^6.17.2
-    ; CircularBuffer by AgileWare
-    1796
-    ; TinyGPSPlus by Mikal Hart
-    1655@1.0.2
+    rlogiacco/CircularBuffer @ ^1.3.3
```
Comment (Halbelunge) [inline platformio.ini:34]: Whats the reason for changing the CircularBuffer Source? Is there any difference in behavior?

Comment (amandel) [reply platformio.ini:34]: This is no change - this is the same lib but not pulling it by numeric Id but the descriptive name.

Comment (Halbelunge) [reply platformio.ini:34]: Ahh nice ok, looked different on the first view if you compare the lib name in the comment and in the link.

Comment (amandel) [reply platformio.ini:34]: Yes you are right.... I did a search for the number in platformIo and it came up with this name.

**Pull request merged**